Customizing shell for Lassen.

The following have been reloaded with a version change:
  1) base-gcc/8.3.1 => base-gcc/11.2.1


Lmod is automatically replacing "clang/ibm-16.0.6-cuda-11.2.0-gcc-8.3.1" with
"gcc/10.2.1".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/rolling-release

Activating 'timing.main' environment for '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/bin/conda'.
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test
Driver directory: /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run
Thu Jan 23 17:59:23 PST 2025
Running parallel timing tests...
* Running prediction-scalability test in scalability_test.
** Running prediction-scalability_np32 on 32 ranks.
Making 3D mesh with the following parameters:
Output name: actii
Number of threads: 1
Mesh size: 9.535
Injector ratio: 2
Sample boundary layer ratio: 4
Bulk boundary layer ratio: 4
Injector boundary layer ratio: 2
Cavity boundary layer ratio: 2
Surround boundary layer ratio: 2
Isolator factor: 2
Cavity factor: 4
Injector factor: 3
Shear layer factor: 4
Sample factor: 4
Make link actii.msh for finished mesh.
Expected mesh file already exists, skipping mesh creation.
++ runoptions='-n 32'
++ [[ ! -z '' ]]
++ jsrun -g 1 -a 1 -n 32 bash /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/emirge/mirgecom/scripts/lassen-parallel-spawner.sh python -u -O -m mpi4py driver.py -c prediction-scalability_np32 -g /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test/log_data -i run_params_np32.yaml --log --lazy
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
Custom casename 'prediction-scalability_np32'
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,151 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
2025-01-23 18:00:08,152 - INFO - grudge.array_context - get_reasonable_array_context_class: MPIFusionContractorArrayContext lazy=True distributed=True device-parallel=True
Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

Using user input from file: run_params_np32.yaml
Running driver.py

/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/wall.py:37: UserWarning: <class 'y3prediction.wall.WallVars'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.wall.WallVars'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:149: UserWarning: <class 'y3prediction.prediction.StepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.StepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py:179: UserWarning: <class 'y3prediction.prediction.WallStepperState'> does not have __array_ufunc__ set. This will cause numpy to attempt broadcasting, in a way that is likely undesired. To avoid this, set __array_ufunc__ = None in <class 'y3prediction.prediction.WallStepperState'>.
  @with_container_arithmetic(bcast_obj_array=False,
2025-01-23 18:00:11,134 - INFO - mirgecom.mpi - Using mpi4py version 4.0.1 with pkl5 scatter support.
2025-01-23 18:00:11,134 - INFO - mirgecom.mpi - mpi4py is using 'Open MPI v10.3.1.03rtm0, package: IBM Spectrum MPI, ident: IBM Spectrum MPI, repo rev: IBM_SPECTRUM_MPI_10.03.01.02_2020.08.03_XAA0, Unreleased developer copy ' with MPI v3.1.
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
2025-01-23 18:00:32,372 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
2025-01-23 18:00:32,372 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,373 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,373 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
2025-01-23 18:00:32,373 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
2025-01-23 18:00:32,373 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
2025-01-23 18:00:32,373 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
2025-01-23 18:00:32,373 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
2025-01-23 18:00:32,374 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    logmgr = initialize_logmgr(True,
               File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
    logmgr = initialize_logmgr(True,
        2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
    logmgr = initialize_logmgr(True,
          File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
     ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
    logmgr = initialize_logmgr(True,
     2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
     ^^^^^^^^^^^^^^^^^^^^^^^
        ^^^^^^^^^^^^^Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,378 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
    logmgr = initialize_logmgr(True,
       Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
    logmgr = initialize_logmgr(True,
    2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
      ^^^^^^^^^^^^^^^^^^^^^^^
    logmgr = initialize_logmgr(True,
          File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
2025-01-23 18:00:32,377 - INFO - mirgecom.logging_quantities - emirge's version.sh: completed (20.17s wall 0.00x CPU)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
         ^^^^^^^^^^^^^^^^^^^^^^^Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
     ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line

  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 214, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 46, in run_command_line
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "driver.py", line 80, in <module>
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    logmgr = initialize_logmgr(True,
             ^^^  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    logmgr = initialize_logmgr(True,
             ^^^^^^subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
             ^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
              main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
             ^^  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 80, in <module>
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
    logmgr = initialize_logmgr(True,
          main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
    logmgr = initialize_logmgr(True,
       File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
    logmgr = initialize_logmgr(True,
             ^^^^    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
        ^^^^^^^^^^^^^^^^    logmgr = initialize_logmgr(True,
     File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
    main(actx_class, restart_filename=restart_filename,
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/mpi.py", line 148, in wrapped_func
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
    logmgr = initialize_logmgr(True,
             ^^^^    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
          ^^^^^^^^^^^^^^^^^^^^^  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/y3prediction/prediction.py", line 1134, in main
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
    logmgr = initialize_logmgr(True,
       File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
    logmgr = initialize_logmgr(True,
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
        ^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 81, in initialize_logmgr
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/logging_quantities.py", line 216, in add_package_versions
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 466, in check_output
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/subprocess.py", line 571, in run
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
subprocess.CalledProcessError: Command '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/version.sh' returned non-zero exit status 1.
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 7 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 6 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 9 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 8 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 18 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 19 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 16 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 11 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 21 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 17 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 20 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 10 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 4 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 1 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 22 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 5 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 23 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 2 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 15 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 3 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 14 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 24 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 25 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 31 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 30 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 29 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 26 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 27 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 28 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 12 in communicator MPI_COMM_WORLD
with errorcode 1.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
++ return_code=1
++ set +x
mv: cannot stat ‘viz_data’: No such file or directory
mv: cannot stat ‘restart_data’: No such file or directory
**  scalability_test/prediction-scalability_np32 failed.
Thu Jan 23 18:00:32 PST 2025
Scaling/timing tests done.
Passing tests: 
Failing tests:  scalability_test/prediction-scalability_np32

------------------------------------------------------------
Sender: LSF System <lsfadmin@lassen710>
Subject: Job 6778113: <scale32> in cluster <lassen> Exited

Job <scale32> was submitted from host <lassen709> by user <mtcampbe> in cluster <lassen> at Thu Jan 23 15:38:12 2025
Job was executed on host(s) <1*lassen710>, in queue <pbatch>, as user <mtcampbe> in cluster <lassen> at Thu Jan 23 17:58:29 2025
                            <40*lassen327>
                            <40*lassen333>
                            <40*lassen664>
                            <40*lassen182>
                            <40*lassen339>
                            <40*lassen185>
                            <40*lassen186>
                            <40*lassen187>
</g/g17/mtcampbe> was used as the home directory.
</p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test> was used as the working directory.
Started at Thu Jan 23 17:58:29 2025
Terminated at Thu Jan 23 18:01:09 2025
Results reported at Thu Jan 23 18:01:09 2025

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
#!/bin/bash
#BSUB -nnodes 8
#BSUB -G uiuc
#BSUB -W 120
#BSUB -J scale32
#BSUB -q pbatch
#BSUB -o scal32.txt

source ../emirge/config/activate_env.sh
source ../emirge/mirgecom/scripts/mirge-testing-env.sh
source ../scripts/multi_scalability.sh -p ../ -s 32 -n 32

------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   2.00 sec.
    Max Memory :                                 59 MB
    Average Memory :                             56.19 MB
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Swap :                                   1634 MB
    Max Processes :                              4
    Max Threads :                                27
    Run time :                                   161 sec.
    Turnaround time :                            8577 sec.

The output (if any) is above this job summary.

